internal/provider/resource_setting_mgmt_test.go

```diff
@@ -39,6 +56,13 @@ func TestAccSettingMgmt_site(t *testing.T) {
 
 func TestAccSettingMgmt_sshKeys(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
```

Comment:
You could move this into a helper function

Reply:
Yeah I think if/when we expand these tests / resources we should just do that to make it a bit easier.